Cargo.toml

```diff
@@ -50,30 +50,30 @@ unicode-width = "0.2.0"
 #
 # NOTE: You can automatically update these dependencies by running ./scripts/update_swc_deps.ts
 swc_atoms = "=7.0.0"
-swc_common = "=14.0.3"
-swc_config = { version = "=3.1.1", optional = true }
+swc_common = "=14.0.4"
+swc_config = { version = "=3.1.2", optional = true }
 swc_config_macro = { version = "=1.0.1", optional = true }
 swc_ecma_ast = { version = "=15.0.0", features = ["serde-impl"] }
-swc_ecma_codegen = { version = "=17.0.0", optional = true }
+swc_ecma_codegen = { version = "=17.0.2", optional = true }
 swc_ecma_codegen_macros = { version = "=2.0.2", optional = true }
 swc_ecma_loader = { version = "=14.0.0", optional = true }
 swc_ecma_lexer = "=23.0.1"
-swc_ecma_parser = "=23.0.0"
-swc_ecma_transforms_base = { version = "=25.0.0", features = ["inline-helpers"], optional = true }
-swc_ecma_transforms_classes = { version = "=25.0.0", optional = true }
-swc_ecma_transforms_compat = { version = "=27.0.0", optional = true }
+swc_ecma_parser = "=24.0.1"
```

Comment:
Is it possible to not bump any swc major versions? Maybe we can only bump the patch for swc_common? If so, that would be preferable because swc upgrades are very involved otherwise in order to make sure everything works properly in all the deno repos. Just bumping the patch in swc_common seems low risk though.